client/blocks/upsell-nudge/index.jsx

```diff
@@ -80,16 +88,35 @@ export const UpsellNudge = ( {
 		return null;
 	}
 
-	if ( ! href && site && ! customerType ) {
-		href = addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
+	// No upsells for WP for Teams sites
+	if ( config.isEnabled( 'signup/wpforteams' ) && siteIsWPForTeams ) {
+		return null;
+	}
+
+	if ( ! href && siteSlug && canUserUpgrade ) {
+		href = addQueryArgs( { feature, plan }, `/plans/${ siteSlug }` );
+		if ( customerType ) {
+			href = `/plans/${ siteSlug }?customerType=${ customerType }`;
+		}
 	}
 
+	const classes = classnames(
+		'upsell-nudge',
+		className,
+		{ 'is-upgrade-blogger': plan && isBloggerPlan( plan ) },
+		{ 'is-upgrade-personal': plan && isPersonalPlan( plan ) },
+		{ 'is-upgrade-premium': plan && isPremiumPlan( plan ) },
+		{ 'is-upgrade-business': plan && isBusinessPlan( plan ) },
+		{ 'is-upgrade-ecommerce': plan && isEcommercePlan( plan ) },
+		{ 'is-jetpack-plan': plan && planMatches( plan, { group: GROUP_JETPACK } ) },
+		{ 'is-wpcom-plan': plan && planMatches( plan, { group: GROUP_WPCOM } ) }
+	);
+
 	return (
 		<Banner
-			callToAction={ preventWidows( callToAction ) }
+			callToAction={ callToAction }
```

Comment:
Seems we added `preventWidows` in #41419 to prevent multi-word buttons breaking in odd places, but I can confirm they still look good here:
<img width="617" alt="Screen Shot 2020-05-14 at 16 21 19" src="https://user-images.githubusercontent.com/1233880/81946015-2220de80-95ff-11ea-9190-2f81d61cf99d.png">

Reply:
Oh, good catch. I moved that logic to `Banner` in #42167 since we should be doing that across the board. :D